docs/solidity-by-example.rst

```diff
@@ -645,4 +645,499 @@ Safe Remote Purchase
 Micropayment Channel
 ********************
 
-To be written.
+Creating and verifying signatures
+=================================
+
+Signatures are used to authorize transactions,
+and they're a general tool that's available to
+smart contracts. In this chapter we'll use it to
+prove to a smart contract that a certain account
+approved a certain message. We'll build a simple
+smart contract that lets me transmit ether, but
+in a unsual way, instead of calling a function myself
+to initiate a payment, I'll let the receiver of
+the payment do that, and therefore pay the transaction
+fee. All this using cryptographics signatures that I'll
+make and send off-chain (e.g. via email). We'll call our
+contract as ReceiverPays, it will works a lot like
+writing a check.Our contract will works like that:
+
+    1. The owner deploys ReceiverPays, attaching enough ether to cover the payments that will be made.
+    2. The owner authorizes a payment by signing a message with their private key.
```

Comment:
`message` sometimes is used synonymously with `transaction`, but I also don't know how a better word. Perhaps offline or off-chain message?